### ISSUE TITLE
Color Contrast Issues on every page of Catalog

### ISSUE THREAD

**Issue by binkylush** [psu-libraries/psulib_blacklight#1382]
@ruthtillman - I want to discuss this further regarding branding and the new psu look and feel for the catalog...
